### ISSUE TITLE
EMTERPRET abort(-12) in 1.38.8 with SDL

### ISSUE THREAD

**Issue by TrevorSundberg** [emscripten-core/emscripten#6818]
It seems we're getting the abort in Emscripten_HandleMouseFocus. Here's the callstack:

Chrome on Windows:
```
ZeroEditor.html:192 -12
printErr @ ZeroEditor.html:192
abort @ ZeroEditor.js:25189
_Emscripten_HandleMouseFocus @ wasm-0d8334fe-23877:18
dynCall_iiii @ wasm-0d8334fe-53881:10
Module.dynCall_iiii @ ZeroEditor.js:24682
handlerFunc @ ZeroEditor.js:7848
jsEventHandler @ ZeroEditor.js:7746
ZeroEditor.html:199 STATUS: Exception Thrown
ZeroEditor.js:25205 Uncaught abort(-12) at Error
    at jsStackTrace (http://127.0.0.1:8080/ZeroEditor.js:1213:13)
    at stackTrace (http://127.0.0.1:8080/ZeroEditor.js:1230:12)
    at abort (http://127.0.0.1:8080/ZeroEditor.js:25199:44)
    at _Emscripten_HandleMouseFocus (wasm-function[23877]:30)
    at dynCall_iiii (wasm-function[53881]:17)
    at Object.Module.dynCall_iiii (http://127.0.0.1:8080/ZeroEditor.js:24682:40)
    at Object.handlerFunc (http://127.0.0.1:8080/ZeroEditor.js:7848:52)
    at HTMLCanvasElement.jsEventHandler (http://127.0.0.1:8080/ZeroEditor.js:7746:24)
This error happened during an emterpreter-async save or load of the stack. Was there non-emterpreted code on the stack during save (which is unallowed)? You may want to adjust EMTERPRETIFY_BLACKLIST, EMTERPRETIFY_WHITELIST.
This is what the stack looked like when we tried to save it: 1,Error
    at Object.handle (http://127.0.0.1:8080/ZeroEditor.js:11137:40)
    at _emscripten_sleep (http://127.0.0.1:8080/ZeroEditor.js:11162:24)
    at emterpret (wasm-function[1295]:16081)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at emterpret (wasm-function[1295]:2450)
    at Object.Module.emterpret (http://127.0.0.1:8080/ZeroEditor.js:24502:37)
    at resume (http://127.0.0.1:8080/ZeroEditor.js:11111:34)
    at http://127.0.0.1:8080/ZeroEditor.js:11165:11
```

We were also getting an issue in SDL_OpenAudioDevice which eventually lead to HandleCaptureProcess and called the abort(-12). After commenting out audio initialization we got further and observed some successful emscripten_sleeps until this exception was thrown.

**Comment by kripken** [comment]
Is `_Emscripten_HandleMouseFocus` (where the assertion is hit) a non-emterpreted function? If so then this may be related to the just-noticed issue #6820

**Comment by TrevorSundberg** [comment]
Indeed `_Emscripten_HandleMouseFocus` was not in my whitelist.

**Comment by kripken** [comment]
Ok, the fix for #6820 should handle it then.
